src/pinns_pde_solve.jl

```diff
@@ -156,8 +156,8 @@ function transform_expression(ex,dict_indvars,dict_depvars, chain,initθ, strate
 end
 
 function get_ε(dim, der_num)
-    epsilon = cbrt(eps(Float32))
-    ε = zeros(Float32, dim)
+    epsilon = cbrt(eps(eltypeθ))
```

Comment:
```suggestion
    epsilon = cbrt(eps(eltype(θ)))
```

Reply:
we do not have access to θ from this method, and the method is executed before training when assembling the model, so there is no need to do dynamic determine the type of θ.

Reply:
got it

Reply:
it's never dynamic BTW if type-stable. It's hoisted to compile time.

Reply:
yes, it is type-stable always,  and it is not necessary to define the type dynamically as we did before in the loss function(`DiffEqBase.parameterless_type(θ)`), it is enough to do it once during compilation.

Reply:
gotcha